tmt/trying.py

```diff
@@ -388,6 +399,28 @@ def action_quit(self, plan: Plan) -> None:
         run_id = click.style(plan.my_run.workdir, fg="magenta")
         self.print(f"Run {run_id} successfully finished. Bye for now!")
 
+    def handle_options(self, plan: Plan) -> None:
+        """Choose requested cli option"""
+
+        for o in self.cli_options:
+            if self.opt(o):
+                getattr(self, f"handle_{o}")(plan)
+
+    def handle_epel(self, plan: Plan) -> None:
+        """ Enable EPEL repository"""
+
+        # tmt run prepare --how feature --epel enabled
+        data: _RawPrepareFeatureStepData = {
+            "name": "prepare-tmt-try-epel",
```

Comment:
I'd drop the "prepare" bit from the name, it already is `prepare` step, no point in marking it as "prepare" once more.

Reply:
seems resolved? pls re-review